lib/dal/src/func/authoring/execute.rs

```diff
@@ -0,0 +1,45 @@
+use telemetry::prelude::warn;
+
+use crate::func::authoring::{FuncAuthoringError, FuncAuthoringResult};
+use crate::func::FuncKind;
+use crate::{AttributePrototype, AttributeValue, DalContext, Func, FuncId};
+
+pub(crate) async fn execute_func(ctx: &DalContext, id: FuncId) -> FuncAuthoringResult<()> {
+    let func = Func::get_by_id_or_error(ctx, id).await?;
+
+    match func.kind {
+        FuncKind::Attribute => update_values_for_func(ctx, &func).await?,
+        FuncKind::Action => {
+            // TODO(nick): fully restore or wait for actions v2. Essentially, we need to run
+            // every prototype using the func id for every component.
+            warn!("skipping action execution...");
+            return Ok(());
+        }
```

Comment:
Here's the thing: do we want to _literally_ create stuff on `save_and_exec`? We should talk about this. That's why it's a warning right now.

Reply:
Yeah, we threw a warning on the front end for create and deletes. I think that's ok because if you're authoring new actions, it seems reasonable that we'd need to do some manual cleanup. Or, if you target localstack, less risk, but either way I think we need this.

Reply:
Good to know! Can you do it on HEAD only or any change set?